insights/parsers/pam.py

```diff
@@ -108,8 +108,8 @@ class PamSSHD(PamDConf):
 """
 
 from collections import namedtuple
-from .. import Parser, get_active_lines, parser
-from ..parsers import unsplit_lines
+from insights import Parser, get_active_lines, parser
+from insights.parsers import unsplit_lines, keyword_search
```

Comment:
OK, personally I'd prefer absolute imports but I'll go with relative as that does seem to be the standard for parsers.